Sources/PackageDescription/Target.swift

```diff
@@ -136,11 +147,19 @@ public final class Target {
     }
     private var _linkerSettings: [LinkerSetting]?
 
+    /// The binary target's checksum.
+    public var _checksum: String? {
```

Comment:
This needs availability annotation.

Reply:
Correct. But we don't know what version to set it at for now.